### PR TITLE
Adding mako cache cleaner upon startup of new version

### PR DESCRIPTION
**Pull request by WebSpider**
Fills FR SiCKRAGETV/sickrage-issues#2712

### CODE REVIEW

File: SickBeard.py [hunk 1]
```diff
@@ -349,6 +349,16 @@ def start(self):
...
 
+        # Clean up after update
+        if sickbeard.GIT_NEWVER:
+            toclean = os.path.join(sickbeard.CACHE_DIR, 'mako')
+            for root, dirs, files in os.walk(toclean, topdown=False):
+                for name in files:
+                    os.remove(os.path.join(root, name))
+                for name in dirs:
+                    os.rmdir(os.path.join(root, name))
+            sickbeard.NEWVER = 0
```
Comment (MGaetan89) [inline SickBeard.py]: GIT_NEWER?

**Comment by miigotu** [comment]
ahaha, wanna PR?

File: SickBeard.py (hunk 1 above)
Comment (WebSpider) [inline SickBeard.py]: Ah. I missed that at 3am :D

Will correct it later today, should be GIT_NEWVER

-----Original Message-----

**Comment by MGaetan89** [comment]
It's set as `int` everywhere. So it needs to be changed everywhere to use `False`properly.

**Comment by MGaetan89** [comment]
https://github.com/SiCKRAGETV/SickRage/pull/2485

**Pull request merged**